changedetectionio/tests/unit/test_notification_diff.py

```diff
@@ -26,6 +26,16 @@ def test_expected_diff_output(self):
         self.assertIn('(removed) for having learned computerese,', output)
         self.assertIn('(removed) I continue to examine bits, bytes and words', output)
 
+        # UTF-8 test, show failure
+        output = diff.render_diff(previous_file=base_dir + "/test-content/before.txt",
+                                  newest_file=base_dir + "/test-content/after-3.txt")
+        output = output.split("\n")
+        self.assertIn('(removed) After twenty years, as cursed as I may be', output)
```

Comment:
this wont do anything useful, shouldnt you be also making sure the utf-8 stuff was removed?

Reply:
yeah... my mistake 
fixed, but I didnt use assertin